src/main/scala/firrtl/util/BackendCompilationUtilities.scala

```diff
@@ -93,8 +89,22 @@ trait BackendCompilationUtilities {
                     cppHarness: File
                   ): ProcessBuilder = {
     val topModule = dutFile
-    val command = Seq("verilator",
-      "--cc", s"$dutFile.v") ++
+
+    val blackBoxVerilogList = {
+      val list_file = new File(dir, firrtl.transforms.BlackBoxSourceHelper.FileListName)
+      if(list_file.exists()) {
+        Seq("-f", list_file.getAbsolutePath)
+      }
+      else {
+        Seq.empty[String]
+      }
+    }
```

Comment:
We can discuss improving in the future, I'm fine with merging this to get stuff working. Hopefully once we find a solution for how to pass data back "up" we can refactor this as well.